pkg/terminal/terminal.go

```diff
@@ -279,6 +289,19 @@ func (t *Term) Run() (int, error) {
 	_, _ = t.client.GetState()
 
 	for {
+		localVars, err := t.client.ListLocalVariables(
```

Comment:
What I was thinking was only clearing the cache here (`locs = nil`) but filling it in the completer function (`if locs == nil { call ListLocalVariables, etc... }`)

Reply:
Sorry, I misunderstood your comment above. You meant a lazy loading of the cache. Seems reasonable, I'm gonna fix it. Thanks.